Themes/default/languages/Reports.english.php

```diff
@@ -129,26 +129,26 @@
 $txt['group_perms_name_send_email_to_members'] = 'Send a forum email to members';
 $txt['group_perms_name_view_mlist'] = 'View the memberlist';
 $txt['group_perms_name_view_stats'] = 'View forum stats';
+$txt['group_perms_name_view_warning_any'] = 'View anyone\'s warning status';
+$txt['group_perms_name_view_warning_own'] = 'View own warning status';
```

Comment:
Nitpicking: Everything else has own before any while this has any before own. Could you maybe change this please?

Reply:
I think I'll do the reverse because I noticed the original strings are all ordered as _any then _own, so i'll do that to the new ones I added as well.

Reply:
sounds good to me